src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import {css} from 'aphrodite';
+import styles from './styles';
+
+const Footer = () => {
+    return (
+      <div id='item_footer' className={css(styles.footer)}>
+        ProjectGreen - Copywright© 2024 - Powered by
+          <a className={css(styles.footer_link)} href='https://github.com/green-devs-test'> Mancogamers</a>
```

Comment:
change this to 
<a className={css(styles.footer_link)} href='https://github.com/green-devs-test'> Green Dev tests</a>

Reply:
Name changed